user-console/source/org/pentaho/mantle/client/ui/tabs/MantleTabPanel.java

```diff
@@ -88,6 +96,7 @@ public void onWindowClosing( ClosingEvent event ) {
       }
     } );
 
+    setIsAsyncModeEnabled();
```

Comment:
Your method is not a setter, so it should be called differently. Also: You are calling the server on every tab that opens, to get a value that is guaranteed to never change (unless you restart the server).

Rename the method to "fetchAsyncModeEnabledStatus()".  This way you can read the constructor and sensibly guess what happens with that method without having to read the method code itself. 

A setXXX method without parameters simply does not match any java-coding style.